codesigning/signing-worker/.gitignore

```diff
@@ -0,0 +1 @@
+natcap-servers-1732552f0202.json
```

Comment:
Does this file get created as part of the signing process?

Reply:
No, it's a private key from google cloud. Part of the ansible playbook copies the key into a known place, so I just wanted to prevent git from tracking it.  I added a comment to the gitignore about this in 3801a351f